specifyweb/frontend/js_src/lib/components/AppResources/Tabs.tsx

```diff
@@ -41,7 +39,7 @@ export function AppResourcesTab({
   resource,
   directory,
   data,
-  isFullScreen: [isFullScreen, handleChangeFullScreen],
+  isFullScreen: [isFullScreen, _],
```

Comment:
does this component ever need `handleChangeFullScreen`? if not, then change the type to remove it from the type (replace `GetSet<boolean>` with `boolean` and then replace usages accordingly)